models/tmp/stg_stripe__customer_tmp.sql

```diff
@@ -1,2 +1,4 @@
 select * 
 from {{ var('customer') }}
+where livemode
+  and not is_deleted
```

Comment:
can these ever be null? might wanna use some coalesces

Reply:
Great catch @fivetran-jamie! There were no null's in our dataset, but always good to leverage coalesce in case there are instances where they are null. 👍

Reply:
Updated to use coalesce for the where clause